tests/python_functional/src/syslog_ng_config/statements/sources/file_source.py

```diff
@@ -28,16 +28,18 @@
 
 
 class FileSource(SourceDriver):
-    def __init__(self, file_name, **options):
+    def __init__(self, file_name=None, **options):
```

Comment:
`config.create_file_source` also defaults `file_name` to `None`. This might be omitted.